src/backend/src/routes/design-review.routes.ts

```diff
@@ -1,5 +1,8 @@
 import express from 'express';
+import DesignReviewController from '../controllers/design-review.controllers';
 
 const designReviewRouter = express.Router();
 
+designReviewRouter.delete('/:drId/delete', DesignReviewController.deleteDesignReview);
```

Comment:
should be :designReviewId, better naming and matches the name you're using in the controller method